.travis.yml

```diff
@@ -73,7 +73,7 @@ matrix:
       before_install: rm -f oc-chef-pedant/Gemfile.lock
       before_script: cd oc-chef-pedant
       script: bundle exec rake chef_zero_spec
-      env: "GEMFILE_MOD=\"gem 'rake'; gem 'chef-zero', github: 'chef/chef-zero'\""
+      env: "GEMFILE_MOD=\"gem 'rake'; gem 'chef-zero', github: 'chef/chef-zero', branch: 'tm/ignore-pedant-universe'\""
```

Comment:
that's in a separate commit so we can just nuke it and then merge. The relevant chef-zero PR is https://github.com/chef/chef-zero/pull/176 .